### PR TITLE
List of peripherals

### PR DESCRIPTION
**Pull request by g-ortuno**
Barebones implementation of a list showing all the peripherals available. The following PRs will be about implementing a proper front end and hooking it up with the other activity. Please review @jyasskin @scheib 

device-2015-03-17-150849

### CODE REVIEW

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/PeripheralsList.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  @Override
+  protected void onListItemClick(ListView l, View v, int position, long id) {
+    super.onListItemClick(l, v, position, id);
+
+    Intent intent = new Intent(this, Peripherals.class);
+    // TODO(g-ortuno): Pass the selected peripheral to the peripheral activity.
+    startActivity(intent);
+  }
+
+  private class PeripheralsListAdapter extends BaseAdapter{
```
Comment (scheib) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/PeripheralsList.java]: Is there a simpler way to have a fixed list that launches an activity?

Comment (g-ortuno) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/PeripheralsList.java]: Done.

**Comment by g-ortuno** [comment]
Ready for review again.

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/PeripheralsList.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+
+
+public class PeripheralsList extends ListActivity {
+
+  // TODO(g-ortuno): Implement heart rate monitor peripheral
+  private static final String[] PERIPHERALS_NAMES = new String[]{"Battery", "Heart Rate Monitor"};
```
Comment (scheib) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/PeripheralsList.java]: Are the two items listed now in the list?

**Comment by scheib** [comment]
LGTM

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/PeripheralsList.java (hunk 2 above)
Comment (g-ortuno) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/PeripheralsList.java]: Yes. But both launch the Battery Peripheral.

**Pull request merged**